Dockerfile

```diff
@@ -162,6 +161,18 @@ RUN mkdir -p /opt && \
 
 ENV PATH $PATH:/opt/gnu-mcu-eclipse/riscv-none-gcc/${RISCV_VERSION}-${RISCV_BUILD}/bin
 
+# Install picolibc Debian package
+ARG PICOLIBC_VERSION=1.4.6-1
```

Comment:
there's a 1.7.3 package, would that work, too?

Reply:
of course!